frontend-react/src/components/DownloadCSVButton.tsx

```diff
@@ -0,0 +1,47 @@
+import { Button, Icon } from "@trussworks/react-uswds";
```

Comment:
I don't think this is a useful abstraction: it's not generic enough for any data (data is typed as `ResponseError[]`), and it's passing in the `csvOptions` anyway.  What'd be more useful in my opinion would be to pull the CSV download into its own function that would could hook up to other buttons as needed.  Will update!